### ISSUE TITLE
StatusInfo not showing at all VS 2015 and 2017

### ISSUE THREAD

**Issue by MartinHT** [lkytal/StatusInfo#1]
I am using StatusInfo (latest version from Visual Studio Gallery) on VS 2015 and VS 2017, but on neither IDE is the information showing on the status bar.
This became evident on VS 2015 a few months ago, but now with VS 2017 I have the same problem.
What can I try?
![2017-03-24_9-38-07](https://cloud.githubusercontent.com/assets/7909929/24286473/ace83418-1075-11e7-8925-4281d7d2afae.png)

**Comment by lkytal** [comment]
Please first update to version 1.2.6 to see whether it works. If not, please let me know  your system version and vs version.

**Comment by MartinHT** [comment]
Yes, I have 1.2.6 installed. 
My setup is:
Windows 10 x64 Enterprise - Spanish
Visual Studio 2017 Enterprise - English
I have attached a snapshot of the VS2017 Options window, showing StatusInfo settings.
![2017-03-24_15-56-09](https://cloud.githubusercontent.com/assets/7909929/24299784/959d67e4-10aa-11e7-8826-0c93e2a1914f.png)

**Comment by lkytal** [comment]
Well it's wired... To locate this problems please try this: change the format string (like add a space between it) and save it, will it show up?

**Comment by MartinHT** [comment]
Sorry, no luck :-(
Here is a little more information:
1) This is NOT a problem on one machine, it is happening on 3 different machines.
2) This is NOT a VS2017 problem, for the last month or so, I have had the same issue with VS2015.
3) I have had this working on VS2015, but it stopped working about 2 months ago.
4) Could there be a conflict with other extensions? I can send you a list if that would help.

**Comment by lkytal** [comment]
Yes it should be helpful. And please check is there any error in activitylog(usually in AppData\Roaming\Microsoft\VisualStudio\15.0_XXX\activitylog.xml)

**Comment by MartinHT** [comment]
No error or warnings are reported in any of the activitylog.xml file.

![2017-03-24_17-02-13](https://cloud.githubusercontent.com/assets/7909929/24302770/0ee69a86-10b4-11e7-9d95-5ff2162af7f7.png)
![2017-03-24_17-03-10](https://cloud.githubusercontent.com/assets/7909929/24302772/0ee9394e-10b4-11e7-9ea6-e04a62eb31db.png)
![2017-03-24_17-03-31](https://cloud.githubusercontent.com/assets/7909929/24302771/0ee76858-10b4-11e7-883f-ee06028eff96.png)
![2017-03-24_17-03-48](https://cloud.githubusercontent.com/assets/7909929/24302773/0ee92940-10b4-11e7-84a9-21387e15ff17.png)
![2017-03-24_15-56-09](https://cloud.githubusercontent.com/assets/7909929/24302774/0f0489ce-10b4-11e7-9ff2-97fc44032c58.png)

**Comment by lkytal** [comment]
Well I cannot reproduce this problem... If you temporarily disable all other third-party extensions and install this [test version](http://lkytal.qiniudn.com/StatusInfo.vsix), will it work?

**Comment by MartinHT** [comment]
Ok, I have downloaded the source code and set up a debugging environment.
It seems that the problem is coming from the line:
TotalCpuCounter = new PerformanceCounter("Processor", "% Processor Time", "_Total");
An exception is thrown:
Cannot load Counter Name data because an invalid index '' was read from the registry.
Upon searching for this error on Google, I found the following answer:
  Click Start, type cmd right click cmd.exe, and select Run as administrator. At the prompt, type lodctr /r and press ENTER.

Apparently, Windows keeps some counters cache in the registry at HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Perflib\009 \ Counter+Help. If this cache becomes corrupted (has blank lines), you get the above errors - and lodctr /r rebuilds that cache.

All is working fine now!

Thanks.

**Comment by lkytal** [comment]
Thanks for your feedback, really unexpected reason.
